gravitee-am-ui/src/app/domain/settings/theme/theme.component.ts

```diff
@@ -305,20 +305,34 @@ export class DomainSettingsThemeComponent implements OnInit {
     });
   }
 
+  private fixAssetUrl(doc: any, tag: string, urlAttribute: string) {
+    const tags = doc.getElementsByTagName(tag);
+    for (let i = 0; i < tags.length; i++) {
+      for (const attribute of tags[i].attributes) {
+        if(attribute.name === urlAttribute){
+          const url = attribute.value;
+          attribute.value = "/" + url;
```

Comment:
This doesn't work when the URL is an external URL to download a logo for example.
You should test if the `attribute.value` start with http before adding the slash.